Products/CMFPlone/interfaces/__init__.py

```diff
@@ -1,47 +1,47 @@
 # -*- coding: utf-8 -*-
 # flake8: noqa
-from basetool import IPloneBaseTool
-from basetool import IPloneCatalogTool
-from basetool import IPloneTool
-from breadcrumbs import IHideFromBreadcrumbs
-from constrains import IConstrainTypes
-from constrains import ISelectableConstrainTypes
-from controlpanel import IControlPanel
-from controlpanel import IDateAndTimeSchema
-from controlpanel import IEditingSchema
-from controlpanel import ILanguageSchema
-from controlpanel import IFilterSchema
-from controlpanel import ILinkSchema
-from controlpanel import ILoginSchema
-from controlpanel import IMaintenanceSchema
-from controlpanel import IMailSchema
-from controlpanel import IMarkupSchema
-from controlpanel import INavigationSchema
-from controlpanel import ISearchSchema
-from controlpanel import ISecuritySchema
-from controlpanel import ISiteSchema
-from controlpanel import ITinyMCELayoutSchema
-from controlpanel import ITinyMCESpellCheckerSchema
-from controlpanel import ITinyMCEResourceTypesSchema
-from controlpanel import ITinyMCEAdvancedSchema
-from controlpanel import ITinyMCESchema
-from controlpanel import ITinyMCEPluginSchema
-from controlpanel import ITypesSchema
-from controlpanel import IUserGroupsSettingsSchema
-from controlpanel import ISocialMediaSchema
-from controlpanel import IImagingSchema
-from controlpanel import IActionSchema
-from controlpanel import INewActionSchema
-from events import IConfigurationChangedEvent
-from events import IReorderedEvent
-from events import ISiteManagerCreatedEvent
-from installable import INonInstallable
-from interface import IInterfaceTool
-from language import ILanguage
-from migration import IMigrationTool
-from patterns import IPatternsSettings
-from password_reset import IPasswordResetToolView
-from password_reset import IPWResetTool
+from Products.CMFPlone.interfaces.basetool import IPloneBaseTool
```

Comment:
This is fine for me. But alternatively we could do this:

```
from .basetool import IPloneBaseTool
```

That is a relative import, but explicit, so that should be fine for all Python versions.
Is there a reason to prefer one over the other? I ask because I don't know!

Reply:
AFAIK some people do not like relative imports and also the Google doesn't.

- https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports

I do like them but I do not want to fight for a silly thing.

Reply:
👎 on relative explicit imports, if we have to be explicit anyway, better do it fully.

This way sorting is no brainer, QA tools have it easier and things look more harmonious 🙂